src/chocolatey/infrastructure/powershell/PoshHostUserInterface.cs

```diff
@@ -129,7 +129,7 @@ public override void WriteProgress(long sourceId, ProgressRecord record)
             }
 
             // http://stackoverflow.com/a/888569/18475
-            Console.Write("\rProgress: {0}% - {1}".format_with(record.PercentComplete.to_string(), record.StatusDescription.PadRight(50, ' ')));
+            Console.Write("\rProgress: {0}% - {1}".format_with(record.PercentComplete.to_string(), record.StatusDescription).PadRight(Console.WindowWidth));

```

Comment:
It's System.Console.WindowWidth.

Docs: https://msdn.microsoft.com/en-us/library/system.console.windowwidth(v=vs.110).aspx
Source: http://referencesource.microsoft.com/mscorlib/a.html#19f2a20d8adf969c

According to the source code it is possible for it to throw an exception if we can't get a console handle, but I haven't been able to produce that state. I tested redirecting to a file from both powershell and cmd. I tested on Powershell 2.0 as well, and I tested Boxstarter, since it captures the choco output.

```
choco install -y 7zip --allow-unofficial > C:\users\bill\desktop\output.log
```

Produces this file:
```
Downloading 7zip.install 64 bit
  from 'http://www.7-zip.org/a/7z1602-x64.exe'

Progress: 6% - Saving 92.41 KB of 1.31 MB (94632/1378550)
Progress: 19% - Saving 266.36 KB of 1.31 MB (272752/1378550)
Progress: 29% - Saving 397.53 KB of 1.31 MB (407072/1378550)
Progress: 52% - Saving 705.5 KB of 1.31 MB (722432/1378550)
Progress: 72% - Saving 980.68 KB of 1.31 MB (1004212/1378550)
Progress: 85% - Saving 1.13 MB of 1.31 MB (1180872/1378550)
Progress: 100% - Saving 1.31 MB of 1.31 MB (1378550/1378550)
Progress: 100% - Completed download of C:\Users\bill\AppData\Local\Temp\chocolatey\7zip.install\16.02\7z1602-x64.exe (1.31 MB).
Download of 7z1602-x64.exe (1.31 MB) completed.
```

Can you think of other scenarios I should test? I suppose we could throw some exception handling around it just in case, but so far I haven't found a scenario where that would be needed.